### PR TITLE
Add CORS

### PR DESCRIPTION
**Pull request by milgner**
The app should be able to deal with it w/o outside assistance.

### CODE REVIEW

**Comment by pke** [comment]
Do we need to add tests for the CORS things?

File: server.go [hunk 1]
```diff
@@ -22,6 +24,28 @@ const (
...
+func mapSlice(src []string, f func(string) string) []string {
+	mapped := make([]string, len(src))
+	for i, v := range src {
+		mapped[i] = f(v)
+	}
+	return mapped
+}
+
+func getCorsOrigins() []string {
+	origins := os.Getenv("KEEDROP_CORS_ORIGINS")
```
[the remark this answers is not in the document]
Comment (milgner) [reply server.go:42]: It's set for the server, contains a comma-separated list of permitted client domains.

**Comment by milgner** [comment]
I thought that since this is an existing library, tests would not be that helpful. But I'll take a look - would certainly be nice to raise coverage a bit

**Comment by milgner** [comment]
Added tests - feel free to merge :)

**Review by pke — APPROVED** [review]

**Pull request merged**